include/class.forms.php

```diff
@@ -1980,6 +1980,24 @@ function getQuickFilterChoices() {
     }
 
     function applyQuickFilter($query, $qf_value, $name=false) {
+        global $thisstaff;
+
+        //special assignment quick filters
+        switch (true) {
+            case ($qf_value == 'assigned'):
+            case ($qf_value == '!assigned'):
+                $result = AssigneeChoiceField::getSearchQ($qf_value, $qf_value);
+                return $query->filter($result);
```

Comment:
Shouldn't this be defined in `AssigneeChoiceField::applyQuickFilter`?

Reply:
We could do that, but we would end up needing to define an applyQuickFilter function for AssignedField, AssigneeChoiceField, and TeamSelectionField. 

applyQuickFilter is hit in class.queue.php getQuery 
$qf->applyQuickFilter()
where $qf could be one of those classes depending on the quick filter.

After discussing it, it looks like it'll be fine to just leave it as is.